src/main/java/com/google/cloud/tools/gradle/appengine/core/DeployExtension.java

```diff
@@ -54,8 +55,10 @@ void setDeployTargetResolver(DeployTargetResolver deployTargetResolver) {
   DeployConfiguration toDeployConfiguration(List<Path> deployables) {
     String processedProjectId = deployTargetResolver.getProject(projectId);
     String processedVersion = deployTargetResolver.getVersion(version);
+
     return DeployConfiguration.builder(deployables)
         .bucket(bucket)
+        .gcloudMode(gcloudMode)
```

Comment:
Safe to pass null or an empty string?

Reply:
yes, all our config options are safe to NULL